src/cli/java/org/commcare/util/screen/EntityScreen.java

```diff
@@ -241,7 +241,27 @@ protected boolean executePendingAction(CommCareSession session) {
      */
     public void updateSelection(String input, @Nullable String[] selectedValues) throws CommCareSessionException {
         setSelectedEntity(input);
+        showDetailScreen();
+    }
+
+    /**
+     * Updates entity selected on the screen
+     *
+     * @param input          input to the entity selected on the screen
+     * @param selectedRefs   references for the selected entity, only contains a single element for the
+     *                       single-select entity screen
+     * @throws CommCareSessionException
+     */
+    public void updateSelection(String input, @Nullable TreeReference[] selectedRefs) throws CommCareSessionException {
```

Comment:
is `selectedRefs` still nullable?

Reply:
no, https://github.com/dimagi/commcare-core/pull/1117/commits/6439bf6568e603226fd9ce776ec26494420a468e